src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using DevChatter.Bot.Core.Commands;
+using DevChatter.Bot.Core.Commands.Operations;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Data.Specifications;
+using DevChatter.Bot.Core.Events.Args;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace UnitTests.Core.Commands.DeleteCommandTests
```

Comment:
Move this class into the `Operations.DeleteCommandOperationTests` folder and namespace